src/In.ProjectEKA.HipService/Consent/Database/Migrations/20200212205522_AddConsentArtefactTable.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace In.ProjectEKA.HipService.Consent.Database.Migrations
+{
+    public partial class AddConsentArtefactTable : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4999696)